src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -14,16 +13,19 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
 import wolox.training.exceptions.responses.BookIdMismatchException;
 import wolox.training.exceptions.responses.BookNotFoundException;
 import wolox.training.models.Book;
 import wolox.training.repositories.BookRepository;
 
-@Controller
+@RestController
 @RequestMapping("/api/training/books")
 public class BookController {
 
-    private static final String MSG = "Book not found";
+    private static final String MSGNOTFOUND = "Book not found";
+    private static final String MSGIDMISMATCH = "Book Id mismatched";
+    private static final String MSGDELETESUCCESSFULLY = "Book Successfully Deleted";
```

Comment:
For clarity remember to add "_" between words (snake_case 😃 ). Just a reminder, I saw you mentioned in a previous PR you would do it 😄